tests/devapps/XamarinDev/XamarinDev/AcquirePage.xaml.cs

```diff
@@ -190,7 +190,7 @@ private async void OnAcquireSilentlyClickedAsync(object sender, EventArgs e)
             }
         }
 
-        private async void OnAcquireClickedAsync(object sender, EventArgs e)
+        private async Task OnAcquireClickedAsync(object sender, EventArgs e)
```

Comment:
Does the app continue to work? I was under the impression that async event handlers cannot use Task. This is the whole reason why we still have `async void` allowed in the language.